lstchain/data/lstchain_standard_config.json

```diff
@@ -276,5 +276,17 @@
     "nsb_tuning": false,
     "nsb_tuning_ratio": 0.52,
     "spe_location": "lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"
+  },
+  "write_interleaved_events":{
```

Comment:
what is the defaults value for `write_interleaved_events`? sorry, could not find it.

Reply:
They are defined here: https://github.com/cta-observatory/ctapipe/blob/0834ca5c0b852ceccd444917d76d8c812d5d5453/ctapipe/io/datawriter.py#L147

Reply:
sorry, I am maybe missing something, but in the link you are giving I do not find the `write_interleaved_events` defined

Reply:
The default values are refined in the DataWriter trailets definition (the target of the above link), which is the Component to be configured by the write_interleaved_events json parameters

Reply:
ok thx!